manifest.json

```diff
@@ -178,6 +178,17 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/fedora/29/helix/amd64",
+              "os": "linux",
+              "tags": {
+                "fedora-29-helix-amd64-$(System:DockerfileGitCommitSha)-$(System:TimeStamp)": {}
```

Comment:
The Docker convention is that `amd64` is implied if the tag doesn't include an architecture.  For consistency reason, please remove from the tag.